resource/abstractions/auditing_rules.md

```diff
@@ -0,0 +1,493 @@
+
+#### ❌ Raises
+* `SecurityRequestError`<br>
+  Raises `SecurityRequestError` when the **SAF Return Code** of a **Security Result** is equal to `4`.
+* `DownstreamFatalError`<br>
+  Raises `DownstreamFatalError` when the **SAF Return Code** of a **Security Result** is greater than `4`.
+* `AlterOperationError`<br>
+  Raises `AlterOperationError` when the **general resource profile** cannot be altered because it does **NOT** exist.
+
+#### 💻 Example
+
+###### Python REPL
+```python
+>>> from pyracf import ResourceAdmin
+>>> resource_admin = ResourceAdmin()
+>>> resource_admin.overwrite_audit_by_access_leve("TESTING","ELIJTEST",alter="success")
+{"step1":{"securityResult":{"resource":{"name":"TESTING","class":"ELIJTEST","operation":"set","requestId":"ResourceRequest","info":["Definition exists. Add command skipped due  to precheck option"],"commands":[{"safReturnCode":0,"returnCode":0,"reasonCode":0,"image":"RALTER  ELIJTEST             (TESTING)  AUDIT( success (ALTER       ))","messages":["ICH11009I RACLISTED PROFILES FOR ELIJTEST WILL NOT REFLECT THE UPDATE(S) UNTIL A SETROPTS REFRESH IS ISSUED."]}]},"returnCode":0,"reasonCode":0}}}
+```
+
+###### Security Result Steps Dictionary as JSON
+```json
+{
+  "step1": {
+    "securityResult": {
+      "resource": {
+        "name": "TESTING",
+        "class": "ELIJTEST",
+        "operation": "set",
+        "requestId": "ResourceRequest",
+        "info": [
+          "Definition exists. Add command skipped due  to precheck option"
+        ],
+        "commands": [
+          {
+            "safReturnCode": 0,
+            "returnCode": 0,
+            "reasonCode": 0,
+            "image": "RALTER  ELIJTEST             (TESTING)  AUDIT( success (ALTER       ))",
+            "messages": [
+              "ICH11009I RACLISTED PROFILES FOR ELIJTEST WILL NOT REFLECT THE UPDATE(S) UNTIL A SETROPTS REFRESH IS ISSUED."
+            ]
+          }
+        ]
+      },
+      "returnCode": 0,
+      "reasonCode": 0
+    }
+  }
+}
+```
+
+## `ResourceAdmin.alter_audit_by_attempt()`
+
+```python
+def alter_audit_by_attempt(
+    self,
+    resource: str,
+    class_name: str,
+    success: Union[str, None] = None,
+    failure: Union[str, None] = None,
+    all: Union[str, None] = None,
+) -> Union[dict, bytes]:
+```
+
+#### 📄 Description
+
+Alter the **Auditing Rules** of a general resource profile specified by **Access Attempt**.
+
+{: .note }
+> _Valid values for **Access Attempts** are limited to the **Access Levels** `alter`, `control`, `read`, and `update`._
+> _You may not specify the same **Access Level** for 2+ **Access Attempts** on a single call of `alter_audit_by_attempt`._
+
+#### 📥 Parameters
+* `resource`<br>
+  The **general resource profile** whose **Auditing Rules** will be altered.
+
+* `class_name`<br>
+  The name of the **class** the specified general resource profile belongs to.
+
+* `success`<br>
+  The **access_level** which **Successes** should be audited for.
```

Comment:
Change ``**access_level**` to `**Access Level**`.

Reply:
changed.